src/minichain/network.py

```
@@ -0,0 +1,1075 @@
+import secrets
+import socket
+import struct
+import time
+from collections import deque
+from dataclasses import asdict, dataclass, field
+from typing import Any, Callable, Coroutine
+
+from minichain.block import Block, BlockHeader
+from minichain.transaction import Transaction
+
+_LOCAL_DISCOVERY_REGISTRY: set["MiniChainNetwork"] = set()
```

Ruff (0.15.2)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)